src/tests/functional/plugin/gpu/concurrency/gpu_concurrency_tests.cpp

```diff
@@ -108,3 +108,166 @@ INSTANTIATE_TEST_SUITE_P(smoke_RemoteTensor, OVConcurrencyTest,
     ::testing::Combine(::testing::ValuesIn(num_streams),
         ::testing::ValuesIn(num_requests)),
     OVConcurrencyTest::getTestCaseName);
+
+TEST(canSwapTensorsBetweenIRs, inputs) {
```

Comment:
Test name is a bit ambiguous. As I understand, IR here means infer request, while common meaning in OV is intermediate representation. So please rename it to something like `canSwapTensorsBetweenInferRequests`